src/main/java/io/ratel/stocktrader/business/stockItem/domain/service/StockItemService.java

```diff
@@ -36,11 +35,11 @@ public class StockItemService{
      * 전체 종목 목록 조회
      * @return 빈 리스트 반환 (추후 데이터 연동 예정)
      */
-    public List<StockItemResponseDto> getAllItems() {
+    public List<StockItemResponse> getAllItems() {
```

Comment:
```suggestion
    public List<StockItem> getAllItems() {
```

StockItem (model) 클래스를 따로 관리하는 것도 방법입니다